src/main/java/com/jungdam/diary/domain/Diary.java

```diff
@@ -77,24 +78,36 @@ public static DiaryBuilder builder() {
         return new DiaryBuilder();
     }
 
-    public Title getTitle() {
-        return title;
+    public Long getAlbumValue() {
+        return this.album.getId();
```

Comment:
```suggestion
        return album.getId();
```
this를 생략한다면 좀 더 간결해 질 것 같다는 생각이 듭니닷. 어떻게 생각하시나요~?

Reply:
넵 전부 지웠슴돠!